src/Contrib/OtlpHttp/OtlpHttpTransportFactory.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\OtlpHttp;
+
+use OpenTelemetry\Contrib\Otlp\OtlpTransportFactoryInterface;
+use OpenTelemetry\SDK\Common\Environment\EnvironmentVariablesTrait;
+use OpenTelemetry\SDK\Common\Export\Http\PsrTransport;
+use OpenTelemetry\SDK\Common\Export\Http\PsrTransportFactory;
+
+class OtlpHttpTransportFactory implements OtlpTransportFactoryInterface
```

Comment:
Should be moved to `OpenTelemetry\Contrib\Otlp`, no need for an additional package.

Reply:
done, which mostly removes the need for `OtlpHttp` (except for http discovery + virtual packages; I've just moved them into suggestions for the time being)